### ISSUE TITLE
Test Navigation: code lab instructions on task 6 ends abruptly

### ISSUE THREAD

**Issue by shepardsn** [google-developer-training/android-basics-kotlin-words-app#287]
**URL of codelab**
https://developer.android.com/codelabs/android-basics-kotlin-test-navigation-components?continue=https%3A%2F%2Fdeveloper.android.com%2Fcourses%2Fpathways%2Fandroid-basics-kotlin-unit-3-pathway-2%23codelab-https%3A%2F%2Fdeveloper.android.com%2Fcodelabs%2Fandroid-basics-kotlin-test-navigation-components#5

**In which task and step of the codelab can this issue be found?**
At the very end of task 6 (after step 6)

**Describe the problem**
The codelab for task 6 ends with the sentence:

"Your test should look something like this:"

but there's nothing else on the page. 

Suggested fix:
- Remove this sentence OR put a complete example of the test method below it.

**Steps to reproduce?**
1. Go to the url, task 6 and scroll to the bottom of the page.

**Versions**
n/a

**Additional information**

**Comment by osuleymanova** [comment]
Duplicate of #255
